src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,10 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { OrdersModule } from '../orders/orders.module';
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule],
```

Comment:
if you're exporting OrdersModule, you need to import it

Reply:
Done